db/migrate/20150312062424_create_poets.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePoets < ActiveRecord::Migration
+  def up
+    create_table :poets do |t|
+      t.string "pen_name", :limit => 45
+      t.string "name", :limit => 45
+      t.string "sex", :limit => 45
+      t.timestamps null: false
```

Comment:
please take out null: false.

update_at can be null for new objects. So with your change we will end up with an error later.